.github/scripts/label_small.sh

```diff
@@ -17,10 +17,10 @@
 LABEL_NAME=small
 MAX_SIZE=100
 
-pr_diff=$(gh pr diff $PR_NUM -R $GITHUB_REPOSITORY)
+pr_diff=$(gh pr view $PR_NUM -R $GITHUB_REPOSITORY --json additions,deletions)
 
-insertions=$(printf "$pr_diff" | grep '^+' | wc -l)
-deletions=$(printf "$pr_diff" | grep '^-' | wc -l)
+insertions=$(echo "$pr_diff" | jq -r '.additions')
```

Comment:
maybe we should use the same terminology here and call this "additions"

Reply:
Done.